README.md

```diff
@@ -100,7 +107,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema contains the data that goes into a database.
```

Comment:
Not quite. Checkout [this analogy](http://stackoverflow.com/a/298751). The file with the data we are putting in our database is the seeds. In production, the data that goes into the database is added by use so we don't seed production databases. We do however need schema for them to describe how to structure out database